easybuild/tools/options.py

```diff
@@ -715,6 +715,28 @@ def postprocess(self):
         if self.options.last_log:
             self.options.terse = True
 
+        # make sure --optarch has a valid format
+        if self.options.optarch:
+            # optarch is in most cases equivalent to strlist, but changing the type breaks compatibility, so we hack it here
```

Comment:
I'm not sure if we should include this comment, since we now have other reasons to take this approach, e.g. `strlist` only supports using `,` as a separator, while we're using `;`